packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -1132,6 +1132,18 @@ const schema: SchemaType<DbPost> = {
     group: formGroups.audio,
     nullable: true
   },
+  forceAllowType3Audio: {
+    type: Boolean,
+    optional: true,
+    hidden: false,
+    defaultValue: false,
+    canRead: ['guests'],
+    canUpdate: ['admins'],
+    canCreate: ['admins'],
+    control: "checkbox",
+    order: 13,
+    group: formGroups.adminOptions,
+  },
```

Comment:
Could you add a comment explaining what this does? Though it has `force` in the name, it's still subject to some constraints